keyboards/keebio/bfo9000/keymaps/deepbluealchemy/keymap.c

```diff
@@ -0,0 +1,250 @@
+  |  F22  |  F23  |  F24  |  TAB  |   Q   |   W   |   E   |   R   |   T   |          |   Y   |   U   |   I   |   O   |   P   |   [   |   ]   |  NOP  |  DEL  |
+  |-------+-------+-------+-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------+-------+-------+-------|
+  |  F19  |  F20  |  F21  |   *   |   A   |   S   |   D   |   F   |   G   |          |   H   |   J   |   K   |   L   |   :   |   "   |   ~   | ENTER |  Home |
+  |-------+-------+-------+-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------+-------+-------+-------|
+  |  F16  |  F17  |  F18  | SHIFT |   \   |   Z   |   X   |   C   |   V   |          |   B   |   N   |   M   |   ,   |   .   |   /   | Shift |   Up  |  End  |
+  |-------+-------+-------+-------+-------+-------+-------+-------+-------|          |-------+-------+-------+-------+-------+-------+-------+-------+-------|
+  |  F13  |  F14  |  F15  |  CTRL |  APP  |  ALT  |  NOP  | SPACE |  NOP  |          |  NOP  | SPACE |  NOP  | AltGr |   FN  |  Ctrl |  Left |  Down | Right |
+  `-----------------------------------------------------------------------'          `-----------------------------------------------------------------------'
+*/
+[_GAME] = LAYOUT(
+  _______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  KC_F22,  KC_F23,  KC_F24,  _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  KC_F19,  KC_F20,  KC_F21,  _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  KC_F16,  KC_F17,  KC_F18,  _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  KC_F13,  KC_F14,  KC_F15,  _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, QWERTY,  _______, _______, _______, _______
+)
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        default_layer_set(1UL << _QWERTY);
+        layer_off (_GIT);
+        layer_off (_GAME);
+      }
+      return false;
+      break;
+    case GIT:
+      if (record->event.pressed) {
+        layer_on(_GIT);
+        layer_off (_GAME);
+      }
+      return false;
+      break;
+    case GAME:
+      if (record->event.pressed) {
+        layer_on(_GAME);
+        layer_off (_GIT);
+      }
+      return false;
+      break;
```

Comment:
These all could be replaced with the `TO` keycode, instead.  It would accomplish the exact same thing (eg, turning off all layers but the destination layer).